---
date: "2016-04-27T00:00:00Z"
external_link: ""
image:
  caption: CiteFuse_schematic
  focal_point: Smart
links:
- icon: newspaper
  icon_pack: far
  name: Manuscript
  url: https://academic.oup.com/bioinformatics/article-abstract/36/14/4137/5827474?redirectedFrom=fulltext
- icon: github
  icon_pack: fab
  name: Github
  url: https://github.com/SydneyBioX/CiteFuse
#slides: example
summary: Single-cell multi-modal analysis of CITE-seq data with CiteFuse
tags:
- Single-cell
- Multi-omics
title: CiteFuse
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""
---

Multi-modal profiling of single cells represents one of the latest technological advancements in molecular biology. Among various single-cell multi-modal strategies, cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) allows simultaneous quantification of two distinct species: RNA and cell-surface proteins. Here, we introduce CiteFuse, a streamlined package consisting of a suite of tools for doublet detection, modality integration, clustering, differential RNA and protein expression analysis, antibody-derived tag evaluation, ligand–receptor interaction analysis and interactive web-based visualization of CITE-seq data.

We demonstrate the capacity of CiteFuse to integrate the two data modalities and its relative advantage against data generated from single-modality profiling using both simulations and real-world CITE-seq data. Furthermore, we illustrate a novel doublet detection method based on a combined index of cell hashing and transcriptome data. Finally, we demonstrate CiteFuse for predicting ligand–receptor interactions by using multi-modal CITE-seq data. Collectively, we demonstrate the utility and effectiveness of CiteFuse for the integrative analysis of transcriptome and epitope profiles from CITE-seq data.

CiteFuse is freely available at http://shiny.maths.usyd.edu.au/CiteFuse/ as an online web service and at https://github.com/SydneyBioX/CiteFuse/ as an R package.
